executorCwl/src/main/scala/dx/executor/cwl/CwlWorkflowExecutor.scala

```diff
@@ -602,8 +603,9 @@ case class CwlWorkflowExecutor(workflow: Workflow,
       (dxExecution, executableLink, execName)
     }
 
-    private def launchCall(stepInputs: Map[DxName, (WorkflowStepInput, (CwlType, CwlValue))],
-                           blockIndex: Int): Map[DxName, ParameterLink] = {
+    private def launchStepCall(stepInputs: Map[DxName, (WorkflowStepInput, (CwlType, CwlValue))],
```

Comment:
Portal function that accepts evaluated step inputs from block/conditional/scatter and checked against target callee's inputs, then `launchCall`.